Merchant/App/Scanner/ScannerViewController.swift

```diff
@@ -22,19 +22,13 @@ class ScannerViewController: UIViewController, AVCaptureMetadataOutputObjectsDel
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         title = Constants.Strings.scanner
-        
-        view.backgroundColor = UIColor.white
-        
-        checkPermission()
+        view.backgroundColor = .black
```

Comment:
@Mattijah **[QUESTION]**
Why did you change the background to black? Moreover, is there a reason why you removed the `checkPermission()` here?

Reply:
**Black background**
- As described in the commit message this change reduces the initial "flash" when switching from the white background to the actual camera view (which is always darker). I'm not a designer, but I do feel like it's more natural like this. I would also suggest to present this view controller modally and not pushing it into a navigation stack. 

**checkPermission**
- This is part of the e5d07a2c776ee692bb15cdb610c1ae64616dd6cb and explained here https://github.com/Bitcoin-com/bitcoin-cash-merchant-ios/pull/3/commits/e5d07a2c776ee692bb15cdb610c1ae64616dd6cb#r290988431